tests/x11/window_system.pm

```diff
@@ -0,0 +1,24 @@
+# X11 regression tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Information about current window system
```

Comment:
Could you add here what value the test provides? That would be great. You might want to add a line with "tags:" as is done in some test modules when you have a fate link or a bug for which you add this information